plugins/email_service/app/javascript/email_service.coffee

```diff
@@ -272,38 +285,38 @@ $(
 #     ccRealAddresses = "sirajudheenam@gmail.com, buzzmesam@gmail.com"
 #     bccRealAddresses = "sirajudheenam@gmail.com, buzzmesam@gmail.com"
 
-#     toFakeAddresses = """ 
-#       rjones@sbcglobal.net, doormat@comcast.net, less mfburgo@me.com, 
+#     toFakeAddresses = """
+#       rjones@sbcglobal.net, doormat@comcast.net, less mfburgo@me.com,
```

Comment:
This is sensible data and It should be removed, even from the git history.

Reply:
This is very important. We are to allowed to even have in the elektra history sensible data.

Reply:
I am not even sure if removing the PR this data stays in the history of elektra. If this is the case you should overwrite the history of the repo.